docs/assets/kubernetes-source.svg

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@binarylogic We need to generate one for kubernetes this currently is using the `docker` source svg.